src/NServiceBus.Persistence.CosmosDB.LogicalOutbox.AcceptanceTests/NServiceBus.Persistence.CosmosDB.LogicalOutbox.AcceptanceTests.csproj

```diff
@@ -17,7 +17,8 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.10.0" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
     <PackageReference Include="NServiceBus.AcceptanceTests.Sources" Version="9.1.1" GeneratePathProperty="true" />
-    <PackageReference Include="Nunit" Version="3.14.0" />
+    <PackageReference Include="NUnit" Version="3.14.0" />
```

Comment:
It was crucial to switch from `Nunit` to `NUnit` making sure the script catches all references.